charts/pgadmin4/templates/ingress.yaml

```diff
@@ -33,9 +36,18 @@ spec:
         paths:
         {{- range .paths }}
           - path: {{ .path }}
+          {{- if and .pathType (semverCompare ">=1.18-0" $kubeVersion) }}
+          pathType: {{ .pathType }}
+          {{- end }}
             backend:
+            {{- if semverCompare ">=1.19-0" $kubeVersion }}
```

Comment:
Missing `service`. Correct config looks like this
```
          - path: /
            pathType: Prefix
            backend:
              service:
                name: pg-pgadmin4
                port:
                  name: http
```
- either `.port.number` or `.port.name` is fine I guess.